src/diffusers/loaders/single_file_model.py

```diff
@@ -356,7 +361,9 @@ def from_single_file(cls, pretrained_model_link_or_path_or_dict: Optional[str] =
             )
 
         else:
-            _, unexpected_keys = model.load_state_dict(diffusers_format_checkpoint, strict=False)
+            _, unexpected_keys = model.load_state_dict(
+                diffusers_format_checkpoint, strict=False, disable_mmap=disable_mmap
```

Comment:
This is torch's load_state_dict method. We can't pass disable_mmap here. 
```suggestion
                diffusers_format_checkpoint, strict=False
```